osbuild/inputs.py

```diff
@@ -88,12 +86,8 @@ def map(self, ip: Input, store: ObjectStore) -> Tuple[str, Dict]:
             }
         }
 
-        with make_args_and_reply_files(store.tmp, args) as (fd_args, fd_reply):
-            fds = [fd_args, fd_reply]
-            client = self.service_manager.start(f"input/{ip.name}", ip.info.path)
-            _, _ = client.call_with_fds("map", {}, fds)
```

Comment:
Fwiw, it seems once this PR is in we could get rid of all fd passing on the host.py level, i.e. client.call_with_fds() and friends because it is only needed it seems for the extra input passing (and returning) (at least right now)